src/lib/common/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function isStringTruthy(str: string): boolean {
```

Comment:
Since `import.meta.env[string]` can be type of `string` or `undefined`, we must account for the case where an environment variable is `undefined` (or even `null` for fun).  By default, we should set `defaultValue` to `false`.

Example 1:
```ts
// When VITE_DEBUG is left undefined, we want debug mode to be turned off.
console.log(isStringTruthy(import.meta.env.VITE_DEBUG, false));

// Output: false
```
Example 2:
```ts
// .env
// VITE_DEBUG=1

// We only want debug mode to be on when it is set to a truthy value in the .env file
// or Vercel build previews.
console.log(isStringTruthy(import.meta.env.VITE_DEBUG, false));

// Output: true
``` 

To get started on improving the usage for catching `undefined` and `null` edge cases,  let's start by refactoring the function signature.

```suggestion
export function isStringTruthy(payload?: string | null, defaultValue = false): boolean {
```

**Note**: The optional argument syntax (`?`) is used in the above example for the `payload` argument to signify that it can be `undefined`. It is shorthand for otherwise saying `(payload: string | undefined | null, ...)`.